SourceLink.Create.CommandLine/SourceLink.Create.CommandLine.csproj

```diff
@@ -7,6 +7,7 @@
     <!-- https://github.com/NuGet/Home/wiki/Adding-nuget-pack-as-a-msbuild-target -->
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <!--<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>-->
+    <DevelopmentDependency>True</DevelopmentDependency>
```

Comment:
It is already here:
https://github.com/ctaggart/SourceLink/blob/master/SourceLink.Create.CommandLine/SourceLink.Create.CommandLine.csproj#L10